server/tree.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path');
+const fs = require('fs');
+
+function isFile(content) {
+  return new Promise((resolve, reject) => {
+    fs.stat(content, (err, stats) => {
+      if (err) {
+        reject(err);
+      }
+
+      resolve(stats.isFile());
+    });
+  });
+}
+
+function formResponse(directoryContent, directoryPath ) {
```

Comment:
extra space before closing parens (probably, result of multiple points edited? same for the next fn)